buddyboss/buddyboss-platform
# PROD-927 #1306 - Fix: Add a confirmation alert before deleting a Profile Filed

### All Submissions:






### Changes proposed in this Pull Request:



Fixes #1306 .

### How to test the changes in this Pull Request:

1. Go to BuddyBoss > Profiles > Profile Fields
2. Create a new fieldset, and fields inside the fieldsets
3. Go to any fieldset and click on Delete of any field
4. Can see the delete confirmation before delete

### Proof Screenshots or Video
https://vimeo.com/624319433/f2b84662a9


### Other information:




### Changelog entry

> Enter a summary of all changes on this Pull Request. This will appear in the changelog if accepted.
